src/marine_eov_broker/MarineRiBroker.py

```diff
@@ -394,51 +401,59 @@ def submit_request(self,
 
         return response
 
-    def submit_sparql_query(self,
-                             query,
-                             endpoint,
-                             eovs,
-                             query_start_date,
-                             query_end_date,
-                             query_min_lon,
-                             query_min_lat,
-                             query_max_lon,
-                             query_max_lat,
-                             output_format,
-                             linked_var,
-                             linked_dataset,) -> list:
-        wrapper = SPARQLWrapper(endpoint)
-        wrapper.setReturnFormat(JSON)
-        wrapper.setQuery(query)
-        results = wrapper.queryAndConvert()
+    def submit_sparql_query(
+        self,
+        query,
+        endpoint,
+        eovs,
+        query_start_date,
+        query_end_date,
+        query_min_lon,
+        query_min_lat,
+        query_max_lon,
+        query_max_lat,
+        output_format,
+        linked_var,
+        linked_dataset,
+    ) -> list:
+        # Build Knowledge source from endpoint
+        #   (could be a list of endpoint, files)
+        kgsource = KGSource.build(endpoint)
+        # Query response as a QueryResult class
+        query_result = kgsource.query(query)
+        # Get the contents of the linked_var as a list
+        content_var = query_result.to_dict()[linked_var]
```

Comment:
this is better --> see previous comment 
pls use `query_result.to_dict()` and not `query_result._data` to reduce dependency-surface